unlock-app/src/components/interface/VerificationStatus.tsx

```diff
@@ -127,7 +143,9 @@ export const VerificationStatus = ({ config }: Props) => {
   )
 
   const invalid = invalidMembership({
-    membershipData: membershipData!,
+    keyId: key!.tokenId.toString(),
+    owner: key!.owner,
+    expiration: key!.expiration,
```

Comment:
Same here. We don't consistent data so I'm directly passing the props after converting.